src/vs/workbench/contrib/notebook/browser/controller/editActions.ts

```diff
@@ -103,6 +103,11 @@ registerAction2(class QuitEditCellAction extends NotebookCellAction {
 						primary: KeyCode.Escape,
 						weight: NOTEBOOK_EDITOR_WIDGET_ACTION_WEIGHT - 5
 					},
+					{
+						when: ContextKeyExpr.and(NOTEBOOK_OUTPUT_FOCUSED, NOTEBOOK_EDITOR_FOCUSED),
```

Comment:
Tested this and it doesn't seem to work as expected:

* When using Voice Over, it says "Escape" to focus on cell container but it doesn't seem to work.
* We might only want to have this keybinding in accessibility mode, otherwise it's very easy to steal normal Escape handling in outputs (e.g., Escape to quit Input's Auto Correction or IME).

Reply:
without the logic to stop event propagation in https://github.com/microsoft/vscode/pull/180715, this extra hotkey isn't really necessary, but _with_ that logic, we want to make it simple to escape from there